sdks/wasm/DebuggerTestSuite/Tests.cs

```diff
@@ -194,6 +194,36 @@ public class SourceList : DebuggerTestBase {
 				}
 			);
 
+		[Fact]
+		public async Task InspectLocalsTypesAtBreakpointSite () =>
+			await CheckInspectLocalsAtBreakpointSite (
+				"dotnet://debugger-test.dll/debugger-test2.cs", 40, 2, "Types",
+				"window.setTimeout(function() { invoke_static_method (\"[debugger-test] Fancy:Types\")(); }, 1);",
+				use_cfo: false,
+				test_fn: (locals) => {
+						CheckNumber (locals, "dPI", Math.PI);
+						CheckNumber (locals, "fPI", (float)Math.PI);
+						CheckNumber (locals, "iMax", int.MaxValue);
+						CheckNumber (locals, "iMin", int.MinValue);
+						CheckNumber (locals, "uiMax", uint.MaxValue);
+						CheckNumber (locals, "uiMin", uint.MinValue);
+
+						CheckNumber (locals, "l", uint.MaxValue * (long)2);
```

Comment:
I think you are incorrect about this, I just tested both `long` and `ulong` with your patch and it seems to be working fine:

<img width="461" alt="Screen Shot 2020-05-05 at 5 58 35 PM" src="https://user-images.githubusercontent.com/346113/81087412-fa908e80-8ef9-11ea-9168-13a45d646e34.png">

Reply:
Sorry, should have looked at the actual values - they are incorrect.